src/networks.json

```diff
@@ -914,6 +914,62 @@
     "start": 536483,
     "imageIPFS": "QmeS75uS7XLR8o8uUzhLRVYPX9vMFf4DXgKxQeCzyy7vM2"
   },
+  "60001": {
+    "key": "60001",
+    "name": "Thinkium Testnet Chain 1",
+    "shortName": "ThinkiumTest1",
+    "chainId": 60001,
+    "network": "thinkiumtest1",
+    "multicall": "0xc49bc485d4f943b287edadbce45eb1a1220ffdfe",
+    "rpc": [
+      "https://test1.thinkiumrpc.net"
+    ],
+    "explorer": "https://test1.thinkiumscan.net/",
+    "start": 323327,
+    "imageIPFS": "QmRfiNT4tDhyxfpYcjNde4BMPPWEAygYNPdAaS9bra6aFC"
+  },
+  "70001": {
+    "key": "70001",
+    "name": "Thinkium Mainnet Chain 1",
+    "shortName": "Thinkium1",
+    "chainId": 70001,
+    "network": "thinkium1",
+    "multicall": "0xc49bc485d4f943b287edadbce45eb1a1220ffdfe",
```

Comment:
Hi @ThinkiumChain Looks good, can you verify and publish all these contracts 
https://chain1.thinkiumscan.net/address/TH66MYS0LG5HQ1XX0FRMZYLDQJVRS5CUJGE
so i can check the multicall

Reply:
Hi, i have register multicall contract in our chain. you'd better user 0x address instead of TH address to use multicall. now please try it.

Reply:
Hi, how about it ?